src/app.js

```diff
@@ -38,61 +68,47 @@ export const start = async args => {
   }
 
   try {
-    const response = await getRepos(username);
-
-    const repositories = response.user.repositories.edges;
+    var userGraph = await getRepos(username);
+  } catch (err) {
+    // This error message from the GitHub API is especially ugly.
+    process.stderr.write(`Error: could not get user ${username}.\n`);
```

Comment:
Let's state in our own message where this error was thrown and then spit out the actual error message because those error logs are still helpful.

```
process.stderr.write(`Error: Could not get repos from user: ${username}.\n`;
process.stderr.write(error);
```